qiskit/opflow/primitive_ops/__init__.py

```diff
@@ -68,5 +68,5 @@
            'CircuitOp',
            'PauliSumOp',
            'TaperedPauliSumOp',
-           'Z2Symmetries'
+           'Z2Symmetries',
```

Comment:
I don't think this change is necessary.

Reply:
Ok, thanks

Reply:
Sorry, do you mean only this line is unnecessary? or all `Z2Symmetries` are in this file unnecessary?

Reply:
The last comma you appended

Reply:
I see. Thank you.

Reply:
This is something I mixed to resolve conflicts when I merged master. I will remove this.